spectator-api/src/main/java/com/netflix/spectator/impl/AtomicDouble.java

```diff
@@ -124,4 +127,9 @@ public void max(double v) {
   @Override public double doubleValue() {
     return get();
   }
+
+  @Override
+  public String toString() {
```

Comment:
Added a `toString` implementation to match the behavior of `AtomicLong` / `AtomicInteger`